tests/Fixtures/app/bootstrap.php

```diff
@@ -23,7 +23,9 @@
 }
 
 // Use the most recent version available
-require end($phpunitInstalls).'/vendor/autoload.php';
+$phpunitLoader = require end($phpunitInstalls).'/vendor/autoload.php';
+$phpunitLoader->unregister();
+$phpunitLoader->register();
```

Comment:
This seems a bit hacky, but it's required in order to not prepend the PHPUnit autoloader, as it is registered before the normal autoloader. So this causes issues specifically with the `Composer\InstalledVersions` class, which is loaded from the `vendor/.phpunit/8.3/vendor` directory instead of the one in the standard `vendor` directory.
So here we need to unregister the autoloader and re-register it so that is it appended to the queue